src/model/chunk.py

```diff
@@ -0,0 +1,36 @@
+## This file contains the function to chunk the input text into smaller pieces for the model to process
+## The input text is tokenized and then split into chunks of size chunk_size
+
+from transformers import AutoTokenizer
+import torch
+
+def chunk(inp: str, 
+          tokenizer: AutoTokenizer, 
+          chunk_size: int = 256, 
+          overlapping_len: int = 3,
+          num_chunks: int = 128) -> list:
```

Comment:
`num_chunks` is not a descriptive variable name, something like `max_chunks` would be better

Reply:
Implemented in https://github.com/MichiganDataScienceTeam/F24-mini-copilot/pull/24/commits/926beedac4e3a431f36582d81dae250ddf5e172f